hosts

```diff
@@ -59,10 +59,14 @@ celery-0.galaxyproject.eu
 # Baremetal
 [galaxyservers]
 sn06.galaxyproject.eu ansible_ssh_user=centos
+sn07.galaxyproject.eu ansible_ssh_user=groot
```

Comment:
What is this user about? This user needs sudo access?

Reply:
That was the local user we created on the host while installing the server. The user can be removed and replaced with a `centos` user like everywhere else (Based on the other hosts, I assume that the `centos` user is the local admin user account?). The user is already given Sudo privileges while installing the host.

Reply:
Yes centos is the local admin account. At some point we should change to rocky (or what ever the new name is), but I would not introduce groot now, but change it either to centos or rocky. Does this make sense?